### ISSUE TITLE
Postman freezes when opening after closing it with a local file showing

### ISSUE THREAD

**Issue by BioeJD** [a85/POSTMan-Chrome-Extension#13288]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

Postman freezes after opening the app, showing loading indicators on the environment and on the last opened request page. I can not use the app again until I delete my entire Roaming folder.

I saw some similar reports of the app freezing, but I have specific steps to reproduce this issue.

### Steps To Reproduce

1. Open Postman
2. Select a POST request that uses a file in its form-data body
3. Select the Body tab for the request
4. For the file value in the body, select a file from the local system that's not in your working directory
5. Close Postman with the page (request body) open
6. Try opening Postman again
7. Postman freezes until you clear the roaming data

### Operating System

Windows

### Postman Version

11.20.0

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by belidzs** [comment]
Can confirm the same issue on the same version. Downgrading to 11.19.0 solves the problem

**Comment by chowwinston** [comment]
@belidzs are you seeing this consistently working in 11.19? We're investigating a similar issue found in 11.12 and I don't believe anything has been fixed since.

**Comment by belidzs** [comment]
@chowwinston seems like you're right.

Even after downgrading I was back at square one after selecting the same request with the upload file, saving, closing then restarting the app.

Maybe downgrading the snap package also restored an earlier state when a different tab was open

**Comment by belidzs** [comment]
I should probably also mention that I'm using the linux snap package and still experience this issue.

However, a quick workaround which seems to work for me (consistently) is to delete the file called `window` from the Postman config directory, then restart the application.

```bash
mv /home/belidzs/snap/postman/291/.config/Postman/window /home/belidzs/snap/postman/291/.config/Postman/window.backup
```

The only thing which got lost was the list of open requests tabs, including the one which triggered this bug.

**Comment by Luminted** [comment]
I am experiencing the same issue.
Version: 11.20.0
OS: Fedora 41

**Comment by MrArabboy** [comment]
I am experiencing the same issue.
Version: 11.21.0
OS: Windows 11

**Comment by chowwinston** [comment]
Hi everyone, this has been fixed in the latest 11.23 release. Please let us know if you're still experiencing this issue. 
Thank you @dev-sharma-08!
